src/bb_godot/tasks.clj

```diff
@@ -17,7 +17,7 @@
 
 (defn home-dir []
   (-> (bb.tasks/shell {:out :string}
-                      "zsh -c 'echo -n ~'")
+                      "sh -c 'printf %s $HOME'")
```

Comment:
i think we can move to `babashka.fs/home` here - it's somewhat recent but should also be cross-platform and a bit cleaner